### DISCUSSION TITLE
Convert PSIG to CuFt and to Liters?

### DISCUSSION THREAD

**Discussion by ChampaignCowboy** [angularsen/UnitsNet#1298] [General]
Is this possible using this tool? We see Pressure, but then we can't see how to convert to Liter which is our ultimate goal.

**Comment by angularsen** [comment]
Hi, yes there is the `Volume` quantity and its units `CubicFoot` and `Liter`. I'm not sure how you meant to convert from pressure to volume?

This is how you create quantities and convert between units:
```cs
var psi = Pressure.FromPoundsForcePerSquareInch(100);
var pa = psi.Pascals; // 689475.7293168361

var v = Volume.FromCubicFeet(100);
var liters = v.Liters; // 2831.6846592

// You can also convert between different pressure references:
var psig = new ReferencePressure(psi, PressureReference.Gauge);
var psia = psig.Absolute; // 114.7 psi
var psiv = psig.Vacuum; // -100 psi
```

Did that answer your question?

**Comment by ChampaignCowboy** [comment]
We will check this next week when we get back. Thanks!!!Mark OrstedOn Aug 4, 2023, at 5:20 PM, Andreas Gullberg Larsen ***@***.***> wrote:﻿
Hi, yes there is the Volume quantity and its units CubicFoot and Liter. I'm not sure how you meant to convert from pressure to volume?
This is how you create quantities and convert between units:
var psi = Pressure.FromPoundsForcePerSquareInch(100);
var pa = psi.Pascals; // 689475.7293168361

var v = Volume.FromCubicFeet(100);
var liters = v.Liters; // 2831.6846592

// You can also convert between different pressure references:
var psig = new ReferencePressure(psi, PressureReference.Gauge);
var psia = psig.Absolute; // 114.7 psi
var psiv = psig.Vacuum; // -100 psi
Did that answer your question?
